app/helpers/application_helper/button/auth_key_pair_cloud_create.rb

```diff
@@ -2,11 +2,8 @@ class ApplicationHelper::Button::AuthKeyPairCloudCreate < ApplicationHelper::But
   def disabled?
     # check that at least one EMS the user has access to supports
     # creation or import of key pairs.
-    Rbac.filtered(ManageIQ::Providers::CloudManager.all).each do |ems|
-      if ems.supports?(:auth_key_pair_create)
-        return false
-      end
-    end
+    return false if Rbac.filtered(ManageIQ::Providers::CloudManager.select(&:supports_auth_key_pair_create)).any?
```

Comment:
That said, for this I could go straight to the model and do `Rbac.filtered(AuthKeyPair.any? { |akp| akp.supports?(:create) })`

Reply:
Yeah, I think it's preferable to go straight to the model and not through the virtual column...IMO the virtual columns should be temporary, so anywhere we can drop them I think is preferable.

Reply:
:+1: yup totally agree